src/components/Footer.tsx

```diff
@@ -48,7 +48,7 @@ export default function Footer() {
             ))}
             </div>
             <div className="mt-8 md:mt-0 md:order-1">
-            <p className="text-center text-base text-gray-400">&copy; 2021 Pablo FELPs Portfolio - Made with Tailwind. </p>
+            <p className="text-center text-base text-gray-400">&copy; 2022 Pablo FELPs Portfolio - Made with Tailwind. </p>
```

Comment:
What do you think to use [new Date()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) and get the years? In this way, you will get the current year every time.

Reply:
Nice one! Thanks for the hint, I will add it right now.